cirq/testing/random_superposition.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+def normalize(array: np.ndarray):
+    norm = np.linalg.norm(array)
+    return array / norm
+
+def random_superposition(size: int) -> np.ndarray:
+    """
+    Args:
+        size: Specified size returns a 2^size length array.
```

Comment:
This convention is opposite to the one we used for `random_unitary`. Take a `dim: int` argument instead. We should keep them consistent.

Reply:
Done.